attachments_component/site/src/Controller/AttachmentsController.php

```diff
@@ -89,9 +90,12 @@ public function displayString($parent_id, $parent_type, $parent_entity,
 		$model = $this->getModel('Attachments', 'Site');
 		if ( !$model ) {
 			$errmsg = Text::_('ATTACH_ERROR_UNABLE_TO_FIND_MODEL') . ' (ERR 60)';
-			throw new \Exception($errmsg, 500);
```

Comment:
Why did you remove the exception here? Did you find a bug that something wasn't handled correctly?

Reply:
hello sorry yes I should put it back